Pseudocode

```diff
@@ -0,0 +1,38 @@
+What sport would you like to get advice on?
+Tennis or quidditch? (T/Q)
```

Comment:
love the pseudo-coding, good job mapping out the control flow. One thing that might be helpful to include is thinking about which segment of program cannot be executed until previous section have completed. This process could help with brainstorming where to include a loop.